test/backend/routes.spec.js

```diff
@@ -294,22 +306,22 @@ describe('Authentication:', () => {
             // This ensures that any requests to plotly fail so that we can catch them
             saveSetting('PLOTLY_API_DOMAIN', 'bad-domain.plot.ly');
 
-            return GET('settings').then(res => res.json().then(json => {
+            return wait(1000).then(() => GET('settings').then(res => res.json().then(json => {
```

Comment:
Any particular reason for adding this wait?

Reply:
so that the token expires.